### PR TITLE
django with landingpage

### PR DESCRIPTION
**Pull request by rebert**

### CODE REVIEW

**Comment by rebert** [comment]
ongoing

**Comment by rebert** [comment]
@ltshb @hansmannj : state conatiner with intial payload ("hello world")

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+
+
+.PHONY: format-lint
+format-lint: format lint
+
+
+# Test target
+
+.PHONY: test
+test: $(REQUIREMENTS_TIMESTAMP)
```
Comment (ltshb) [inline Makefile:108]: test should depends on $(DEV_REQUIREMENTS_TIMESTAMP) otherwise nose2 might be missing

File: project/app/views.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from django.http import JsonResponse, HttpResponse
```
Comment (ltshb) [inline project/app/views.py:1]: Import should be on separate lines: https://github.com/geoadmin/doc-guidelines/blob/master/PYTHON.md#5-imports-formatting

File: project/project/settings.py [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': BASE_DIR / 'db.sqlite3',
+    },
+    'dev': {
+         'ENGINE': 'django.contrib.gis.db.backends.postgis',
+         'NAME': 'geodjango',
+         'USER': os.getenv('DB_USER', 'geo'),
+         'USER': os.getenv('DB_PW', 'very_secret'),
```
Comment (ltshb) [inline project/project/settings.py:89]: Should not be PW or PASSWORD here ?

**Review by ltshb — COMMENTED** [review]
Good I would suggest to rename the PR (remove the WIP ;-)) and enventually some small changes see comments

**Review by ltshb — APPROVED** [review]
Nice :-)

**Comment by hansmannj** [comment]
Nice, Thanks Tobias! :+1:

File: project/project/urls.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.contrib import admin
+from django.urls import path, include
+
+urlpatterns = [
+    path('api/stac/v0.9/api.html', include('app.urls')),
```
Comment (hansmannj) [inline project/project/urls.py:20]: A small thing: I am not sure, if this should only be 'api/stac/v0.9/' without the 'api.html'

**Pull request merged**